samples/CommunityToolkit.Maui.Sample/AppShell.xaml.cs

```diff
@@ -172,7 +172,7 @@ public static string GetPageRoute(Type viewModelType)
 		return uri.Uri.OriginalString[..^1];
 	}
 
-	static string GetPageRoute(Type galleryPageType, Type contentPageType) => $"/{galleryPageType.Name}/{contentPageType.Name}";
+	static string GetPageRoute(Type galleryPageType, Type contentPageType) => $"//{galleryPageType.Name}//{contentPageType.Name}";
```

Comment:
Do we need the second slash between } and {. I think there was only one before the original PR broke it?